src/lambda/handler-runner/docker-runner/DockerRuntime.js

```diff
@@ -0,0 +1,33 @@
+import { supportedRuntimesArquitecture } from "../../config/index.js"
```

Comment:
It does not run because of this line, it should be changed to

`import { supportedRuntimesArquitecture } from "../../../config/supportedRuntimes.js"`

Reply:
I just changed this, sorry for the issue.